### DISCUSSION TITLE
Semi colon syntax error in Flatlist of react-native-reanimated library

### DISCUSSION THREAD

**Discussion by dsound-zz** [software-mansion/react-native-reanimated#5302] [Q&A]
I cannot get react-native-reanimated to build. There is an error with in the library itself. 

https://imgur.com/a/iBSmrcB

I see there are similar reports of this error in bug reports here:

https://github.com/software-mansion/react-native-reanimated/issues/4645

Any thoughts on what to do? The library is unusable til it's fixed. 

react-native: 0.67.5
react-native-reanimated: 3.5.4

**Comment by MatiPl01** [comment]
I think I can safely close this discussion. The latest reanimated version doesn't seem to have this issue anymore so it must have been resolved before.

Feel free to re-open the discussion if you still need help.
